nautobot/extras/models/models.py

```diff
@@ -63,7 +63,7 @@ def _validate_with_schema(self, data_field, schema_field):
                 raise ValidationError({data_field: [f"Validation using the JSON Schema {schema} failed.", e.message]})
 
 
-@extras_features("graphql")
+@extras_features("graphql", "dynamic_groups")
```

Comment:
I don't *think* this is needed - this would be to make it possible to create a DynamicGroup *of* ConfigContext objects.

Reply:
I can't think of any reason this would be used, but I'll leave it here for a bit and see if there's any other feedback on this item.

Reply:
Oh, I misread this. My intent was to be able to apply config contexts by using the metadata in the config context to point to a dynamic group. Maybe I'm barking up the wrong tree here. 

i.e.
``` 
---
_metadata:
  name: "Datacenter -> Arista: Management Settings"
  weight: 600
  description: "Device Management settings for Datacenter Arista Devices"
  is_active: true
  dynamic_group:
    - slug: datacenter-arista
```

Reply:
Ah, good thought! You covered that I think with the change [here](https://github.com/nautobot/nautobot/pull/3182/files#diff-560821d3266ff4bf93a11d6a8ba0dd831bc4417a619607845749954159b07259R547). Though it would be good to also touch [this list](https://github.com/nautobot/nautobot/blob/develop/nautobot/extras/datasources/git.py#L398) as well so that we can support a git repo with contexts in `config_contexts/dynamic_groups/<slug>.json` too.

Reply:
Okay, I added that!

Reply:
Let's remove this line, I think?

Reply:
Actually, I'm thinking this is needed for the functionality described above. When I first did my test that was the only thing I updated and it associated the config context to the devices in the DG. Maybe I'm missing something?

Reply:
This line should go away! :)

Reply:
Removed!